config/kubernetes/sig-release/teams.yaml

```diff
@@ -329,59 +329,60 @@ teams:
             description: Members of the CI Signal team for the current release
               cycle.
             members:
+            - mansikulkarni96 # 1.29 CI Signal Shadow
+            - pacoxu # 1.29 CI Signal Shadow
+            - reetasingh # 1.29 CI Signal Shadow
+            - rjsadow # 1.29 CI Signal Shadow
+            - sontek # 1.29 CI Signal Shadow
             - Vyom-Yadav # 1.29 CI Signal Lead
-            #- X # 1.29 CI Signal Shadow
-            #- X # 1.29 CI Signal Shadow
-            #- X # 1.29 CI Signal Shadow
-            #- X # 1.29 CI Signal Shadow
             privacy: closed
           release-team-docs:
             description: Members of the Docs team for the current release cycle.
             members:
+            - drewhagen # 1.29 Docs Shadow
+            - harshitasao # 1.29 Docs Shadow
             - katcosgrove # 1.29 Docs Lead
-            #- X # 1.29 Docs Shadow
-            #- X # 1.29 Docs Shadow
-            #- X # 1.29 Docs Shadow
-            #- X # 1.29 Docs Shadow
+            - Princesso # 1.29 Docs Shadow
+            - taniaduggal # 1.29 Docs Shadow
             privacy: closed
           release-team-bug-triage:
             description: Members of the Bug Triage team for the current release cycle.
             members:
+            - bradmccoydev # 1.29 Bug Triage Shadow
             - hailkomputer # 1.29 Bug Triage Lead
-            #- X # 1.29 Bug Triage Shadow
-            #- X # 1.29 Bug Triage Shadow
-            #- X # 1.29 Bug Triage Shadow
-            #- X # 1.29 Bug Triage Shadow
+            - MaryamTavakkoli # 1.29 Bug Triage Shadow
+            - moficodes # 1.29 Bug Triage Shadow
+            - zelenushechka # 1.29 Bug Triage Shadow
             privacy: closed
           release-team-comms:
             description: Members of the Comms team for the current release cycle.
             members:
+            - Abbie # 1.29 Comms Shadow
```

Comment:
@Priyankasaggu11929  The correct github user for Abigail McCarthy is @a-mccarthy

Reply:
Hey @krol3 -- actually I realised I was using the wrong GitHub username ^. Will correct it. Thanks for the PR though! :slightly_smiling_face: